sdk/appconfiguration/ci.mgmt.yml

```diff
@@ -0,0 +1,34 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
+  branches:
+    include:
+      - main
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - sdk/appconfiguration/ci.mgmt.yml
+      - sdk/appconfiguration/arm-appconfiguration/
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+    exclude:
+      - feature/v4
+  paths:
+    include:
+      - sdk/appconfiguration/ci.mgmt.yml
+      - sdk/appconfiguration/arm-appconfiguration/
+extends:
```

Comment:
nit: Please add an newline before extends and delete the empty line at the end of this file.

I assume you wrote a script to generate these and so if it easy enough to rerun with those update please do.

Reply:
I have updated the format.